app/models/shopping_cart/delivery.rb

```diff
@@ -0,0 +1,7 @@
+module ShoppingCart
+  class Delivery < ApplicationRecord
+    has_many :orders, class_name: 'ShoppingCart::Order'
```

Comment:
if you will use rubocop it will be recommended replace with
```ruby
has_many :orders, class_name: ShoppingCart::Order.name
```
Also for each other class_name attribute

Try
```ruby
rubocop -a
```